beast/tools/calculate_depth.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+from astropy.table import Table, vstack
+from beast.physicsmodel.grid import SEDGrid
+import beast.observationmodel.noisemodel.generic_noisemodel as noisemodel
+from beast.observationmodel.vega import Vega
+
+
+def calculate_depth(
```

Comment:
Would there be a more descriptive name possible?  Something that includes the completeness or something like that?  Just would be better to have the name a bit more descriptive.

Reply:
Can do!